src/wyzeapy/wyze_auth_lib.py

```diff
@@ -132,21 +148,24 @@ async def get_token_with_2fa(self, verification_code) -> Token:
             'https://auth-prod.api.wyze.com/user/login',
             headers=headers, json=payload)
 
-        self.token = Token(response_json['access_token'], response_json['refresh_token'], time.time(), True)
+        self.token = Token(response_json['access_token'], response_json['refresh_token'])
+        self.token_callback(self.token)
         return self.token
 
     @property
-    def should_refresh(self) -> bool:
-        return time.time() - self.token.last_login_time > 59 * 60 * 60
+    async def should_refresh(self) -> bool:
```

Comment:
It looks like this pesky async came back

Reply:
This should be resolved in a later commit.